docs/release-notes.rst

```diff
@@ -65,8 +65,13 @@ For each release, we list the following types of change (in this order):
 - **Development Changes**: Significant changes to Bionic's development process, such
   as changes to our Pytest configuration or our Continuous Integration ("CI").
 
-.. Upcoming Version (Not Yet Released)
-.. -----------------------------------
+Upcoming Version (Not Yet Released)
+-----------------------------------
+
+Improvements
+............
+
+- Add a `should_persist` flag to the SVG mouse hover tooltip of `flow.render_dag()`
```

Comment:
These notes are intended for the user, so we should describe the improvement in terms of what they'll see. So in this case we might write "Added a line to the SVG DAG visualization tooltips indicating whether an entity is persisted." (Instead of mentioning `should_persist`, which is an internal variable name.)

Reply:
That makes a lot of sense. Thanks for explaining all these things!